spec/asyncapi.md

```diff
@@ -333,8 +335,7 @@ Field Name | Type | Description
 
 This object MAY be extended with [Specification Extensions](#specificationExtensions).
 
-##### Contact Object Example
```

Comment:
I don't recall why we removed this one 🤔

Reply:
Oh, that was by mistake I guess. Will revert it.

Reply:
done through b24f71d